tunnel.go

```diff
@@ -407,6 +429,33 @@ func (api *API) GetTunnelConfiguration(ctx context.Context, params GetTunnelConf
 	return tunnelDetails, nil
 }
 
+// TunnelConnections gets all connections on a tunnel.
+//
+// API reference: https://api.cloudflare.com/#cloudflare-tunnel-clean-up-cloudflare-tunnel-connections
+func (api *API) TunnelConnections(ctx context.Context, params TunnelConnectionParams) ([]Connection, error) {
+	if params.AccountID == "" {
+		return []Connection{}, ErrMissingAccountID
+	}
+
+	if params.ID == "" {
+		return []Connection{}, ErrMissingTunnelID
+	}
+
+	uri := fmt.Sprintf("/accounts/%s/cfd_tunnel/%s/connections", params.AccountID, params.ID)
+
+	res, err := api.makeRequestContext(ctx, http.MethodGet, uri, nil)
+	if err != nil {
+		return []Connection{}, err
+	}
+
+	var argoDetailsResponse TunnelConnectionResponse
+	err = json.Unmarshal(res, &argoDetailsResponse)
+	if err != nil {
+		return []Connection{}, errors.Wrap(err, errUnmarshalError)
```

Comment:
you shouldn't use `errors.Wrap`, use `fmt.Errorf` instead.

Reply:
Ah! seems like this change was merged yesterday for all other components. I will rebase to the upstream master and fix this.